languages/python/oso/polar/polar.py

```diff
@@ -196,6 +196,20 @@ def query_rule(self, name, *args, **kwargs):
         """
         return self.query(Predicate(name=name, args=args), **kwargs)
 
+    def query_rule_once(self, name, *args, **kwargs):
```

Comment:
Don't love this name -- very open to alternatives. It seems like a useful method (I believe I've seen folks ask about something similar in slack)